common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2795,6 +2860,71 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("30s")
 
+  val WORKER_HTTP_AUTH_SUPPORTED_SCHEMES: ConfigEntry[Seq[String]] =
+    buildConf("celeborn.worker.http.auth.supportedSchemes")
+      .categories("worker")
+      .version("0.6.0")
+      .doc("A comma-separated list of worker http auth supported schemes." +
+        "<ul>" +
+        " <li>SPNEGO: Kerberos/GSSAPI authentication.</li>" +
+        " <li>BASIC: User-defined password authentication, the concreted implementation is" +
+        " configurable via `celeborn.master.http.auth.basic.provider`.</li>" +
```

Comment:
some places wrongly refer to `master` configurations
```suggestion
        " configurable via `celeborn.worker.http.auth.basic.provider`.</li>" +
```

Reply:
Done